Code/LauncherUnified/Launcher.cpp

```diff
@@ -562,9 +562,11 @@ namespace O3DELauncher
 
         ReturnCode status = ReturnCode::Success;
 
+#ifdef CARBONATED
```

Comment:
Is this an intentional change?

Reply:
Yes. Otherwise the AutomatedTesting project won't compile because this functions call an ebus which was added by carbonated inside a #define